sdk/storage/azure-storage-queue/azure/storage/queue/aio/_models.py

```diff
@@ -15,16 +17,37 @@
 from .._models import QueueMessage, QueueProperties
 
 
+async def async_queue_msg_generator(messages: Any):
+        for q in messages:
+            yield QueueMessage._from_generated(q)
+
+async def async_queue_items_generator(items: Any):
+        for q in items:
+            yield QueueProperties._from_generated(q)
```

Comment:
`azure\storage\queue\aio\_models.py:140: error: Incompatible types in assignment (expression has type "List[QueueProperties]", variable has type "AsyncIterator[Any]")  [assignment]`

Reply:
Removing comment as this doesn't even work properly!